frontend/components/ProfilePublication/Publication.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+
+class Publication extends Component {
+  render() {
+    return (
+      <>
+        {/* {style} */}
+        <a
+          href="{this.props.data.link}"
+          style={{
+            fontSize: 17,
+            textDecorationLine: 'underline',
+            display: 'inline-block',
+            marginTop: '0.5em'
+          }}
+        >
+          {this.props.data.title}
+        </a>
+        <p style={{ display: 'block', marginBlockStart: '0em' }}>
+          {this.props.data.desc}
+        </p>
+        {/* [
+        {
+          publicationName: '1st publication',
+          description: 'about the publication one'
+        },
+
+        {
+          publicationName: '2nd publication',
+          description: 'about the publication two'
+        }
+      ] */}
+      </>
+    )
+  }
+}
+
+// const style = (
+//   <style jsx="true">{`
+//     .publication {
+//       font-size: 30px;
+//     }
+//   `}</style>
+// )
```

Comment:
try to remove commented code. If you need to see the code, you could still see it in the git history

Reply:
yes please its a form of tech debt to have large code blocks commented out because new devs won't know what to do with it and it adds noise

Reply:
removed